src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -148,12 +149,21 @@
 
   public static final String SCHEMA_PATTERN_CONFIG = "schema.pattern";
   private static final String SCHEMA_PATTERN_DOC =
-      "Schema pattern to fetch tables metadata from the database:\n"
+      "Schema pattern to fetch table metadata from the database:\n"
       + "  * \"\" retrieves those without a schema,"
-      + "  * null (default) means that the schema name should not be used to narrow the search, "
-      + "all tables "
-      + "metadata would be fetched, regardless their schema.";
+      + "  * null (default) means that the schema name should not be used to narrow the search,"
+      + " so that all table metadata would be fetched, regardless their schema.";
   private static final String SCHEMA_PATTERN_DISPLAY = "Schema pattern";
+  public static final String SCHEMA_PATTERN_DEFAULT = null;
+
+  public static final String CATALOG_PATTERN_CONFIG = "catalog.pattern";
+  private static final String CATALOG_PATTERN_DOC =
+      "Catalog pattern to fetch table metadata from the database:\n"
+      + "  * \"\" retrieves those without a catalog,"
+      + "  * null (default) means that the catalog name should not be used to narrow the search"
+      + " so that all table metadata would be fetched, regardless their catalog.";
+  private static final String CATALOG_PATTERN_DISPLAY = "Schema pattern";
+  public static final String CATALOG_PATTERN_DEFAULT = null;
```

Comment:
Some DBMSes like Sybase and SQL Server use the catalog for the database and the schema for the owner. That means that to support these DBMSes (and others like them) we have to allow users to be able to specify a catalog pattern.